manifests/rules/nfs3.pp

```diff
@@ -2,6 +2,7 @@
 class nftables::rules::nfs3 {
   nftables::rule {
     'default_in-nfs3':
-      content => 'meta l4proto { tcp, udp } th dport nfs accept comment "Accept NFS3"',
+      # To support nftables >= 0.9.0, but < 0.9.3, `@th,16,16 2049` is used instead of `th dport nfs`
+      content => 'meta l4proto { tcp, udp } @th,16,16 2049 accept comment "Accept NFS3"',
```

Comment:
Can we put this as parameter to the class and then set especially for the particular Debian versions.

Am always pretty nervous about changing people's rules.

Steve.